cloudinit/config/schemas/schema-cloud-config-v1.json

```
@@ -188,6 +188,21 @@
       },
       "minProperties": 1
     },
+    "cc_autoinstall": {
+      "type": "object",
+      "properties": {
+        "autoinstall": {
+          "description": "Opaque autoinstall schema definition for Ubuntu autoinstall. Full schema processed by live-installer. See: https://ubuntu.com/server/docs/install/autoinstall-reference",
+          "type": "object",
+          "properties": {
+            "version": {
+              "type": "integer"
+            }
```

Nit: I know `"additionalProperties": true` is the default behavior, but do you think would it add value to explicitly signal that the intention of this schema is to contain more props than `version` ?

```suggestion
            },
            "additionalProperties": true
```

---

This is a good suggestion +1.

---

Setting `true` here doesn't negatively affect anything, but I'm not sure that it's supported either. https://json-schema.org/understanding-json-schema/reference/object.html#additional-properties says false or a list of additional keys.